pkg/download/classic/downloader.go

```diff
@@ -42,9 +41,11 @@ type (
 	Downloader struct {
 		apisToDownload api.APIs
 
+		filter bool
```

Comment:
please ensure the `NewDownloader` constructor sets this to `true` by default, before applying any options. We also assume the feature flags are ON by default in this case.

Reply:
if that's the reasonable default, you could turn it around, and change it to `skipFilter bool`